exporters/sender/okhttp/src/main/java/io/opentelemetry/exporter/sender/okhttp/internal/OkHttpUtil.java

```diff
@@ -18,6 +18,9 @@
  * at any time.
  */
 public final class OkHttpUtil {
+  // For testing purposes
+  @SuppressWarnings("NonFinalStaticField")
+  static boolean propagateContextInDispatcher = false;
```

Comment:
Can we expose this through a static helper method instead? Even for tests, I don't think mutating static class fields is a habit we want to get into.

Reply:
Got it, it's updated now.